apps/frontend/public/static/export/template.html

```diff
@@ -1,271 +1,638 @@
+                <div
+                  class="hidden w-28 grid-flow-col items-center rounded border border-gray-300 bg-gray-100 px-2 py-1 text-black lg:grid"
+                >
+                  <span>{{{controlSeverity.icon}}}</span>
+                  <span>{{controlSeverity.severity}}</span>
                 </div>
+                <!-- Control title -->
+                <span class="w-1/2 break-all px-6 sm:break-words">{{hdf.wraps.title}}</span>
+                <!-- Control tags -->
+                <div
+                  class="hidden grid-cols-1 gap-x-6 gap-y-2 lg:grid xl:grid-cols-2"
+                >
+                  {{#controlTags}}
+                  <div class="w-24">
+                    <span
+                      class="rounded border border-gray-300 bg-gray-100 px-2 py-1 text-black"
+                      >{{.}}</span
+                    >
+                  </div>
+                  {{/controlTags}}
+                </div>
+
+                <span
+                  class="-mr-1 ml-auto h-5 w-5 shrink-0 rotate-[-180deg] fill-[#336dec] transition-transform duration-200 ease-in-out group-[[data-te-collapse-collapsed]]:mr-0 group-[[data-te-collapse-collapsed]]:rotate-0 group-[[data-te-collapse-collapsed]]:fill-[#212529] motion-reduce:transition-none"
+                >
+                  <svg
+                    xmlns="http://www.w3.org/2000/svg"
+                    fill="none"
+                    viewBox="0 0 24 24"
+                    stroke-width="1.5"
+                    stroke="currentColor"
+                    class="h-6 w-6"
+                    role="img"
+                    title="An arrow indicating accordion collapse state"
+                    aria-label="An arrow indicating accordion collapse state"
+                  >
+                    <path
+                      stroke-linecap="round"
+                      stroke-linejoin="round"
+                      d="M19.5 8.25l-7.5 7.5-7.5-7.5"
+                    />
+                  </svg>
+                </span>
+              </button>
+            </div>
+
+            <!-- Accordion body def -->
+            <div
+              class="multi-collapse-inner px-5 py-4"
+              id="control-accordion-body-{{controlID}}"
+              data-te-collapse-item
+              data-te-collapse-show
+              aria-labelledby="control-accordion-header-{{controlID}}"
+            >
+              <!-- Control description -->
+              {{data.desc}}
+              <hr class="my-3" />
+
+              <!-- Control tables -->
+              <div>
+                <!-- Tests -->
+                <h5 class="text-lg font-bold">Test Results</h5>
+                <table
+                  class="table min-w-full text-left"
+                  aria-describedby="Control test description and result"
+                >
+                  <!-- Generate all available test results -->
+                  {{#hdf.segments}}
+                  <thead>
```

Comment:
maybe instead of using a table header, we can play around with spacing / adding a vertical divider bar.  'name' and 'value' don't really add a lot of clarification to the content of the table so it's basically not worthwhile even having it imo (other than maybe have it be hidden so that it's still there for accessibility purposes idk).

Reply:
Changed as requested.